contracts/core/Farm.sol

```diff
@@ -305,7 +310,7 @@ contract Farm is ICHAOS, IFarm, Ownable {
 
         // Interactions
         if (_pendingRewards != 0) {
-            CHAOS_TOKEN.safeTransfer(to, _pendingRewards);
+            // CHAOS_TOKEN.safeTransferFrom(_self, to, _pendingRewards);
```

Comment:
This change is disabling the Harvest function, how is it possible that the tests are still passing ? :thinking: